code/config.ss

```diff
@@ -179,6 +179,13 @@
       (difficulty . 4)
       (topics string cipher))
 
+     ((slug . affine-cipher)
+      (uuid . "c8969b0a-232a-4c88-ab2a-0047e8b4b6d0")
+      (core . #f)
+      (unlocked-by . anagram)
+      (difficulty . 6)
```

Comment:
I had to re-implement a lot of things for the repo version of this.  Using srfi's it shouldn't require as muh, but it is still quite a bit more involved than either the rotational cipher or the atbash cipher.

If someone decides not to make two maps, then they will have a hell of a ime implementing the decoding function.